### PR TITLE
Dera development

### PR DESCRIPTION
**Pull request by derachimso**
It is still failing and will not run a dev server. But the html seems correct

### CODE REVIEW

**Comment by derachimso** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fgoogleinterns%2Fstep96-2020%2Fpull%2F1) for more info**.

@googlebot I signed it!

File: capstone/target/step-96-2020-1/script.js [hunk 1]
```diff
@@ -0,0 +1,470 @@
+// import $ from 'jquery'
+// import Popper from 'popper.js'
+// import Util from './util'
```
Comment (zhanche-cp) [inline capstone/target/step-96-2020-1/script.js:3]: This file seems mostly commented out. Why do we still need this?

File: capstone/target/step-96-2020-1/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    
+
+      <!-- <main role="main" class="inner cover"> -->
+        <h1 class="cover-heading">Cover your page.</h1>
+        <p class="lead">Cover is a one-page template for building simple and beautiful home pages. Download, edit the text, and add your own fullscreen background photo to make it your own.</p>
+        <p class="lead">
+          <a href="#" class="btn btn-lg btn-secondary">Learn more</a>
+        </p>
+     
+
```
Comment (zhanche-cp) [inline capstone/target/step-96-2020-1/index.html:27]: Can we remove the blank lines here?

File: capstone/target/step-96-2020-1/home.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            <a class="nav-link active" href="#">Home</a>
+            <a class="nav-link" href="#">Features</a>
+            <a class="nav-link" href="#">Contact</a>
+          </nav>
+        </div>
+      </header>
+
+      <main role="main" class="inner cover">
+        <h1 class="cover-heading">Cover your page.</h1>
+        <p class="lead">Cover is a one-page template for building simple and beautiful home pages. Download, edit the text, and add your own fullscreen background photo to make it your own.</p>
```
Comment (zhanche-cp) [inline capstone/target/step-96-2020-1/home.html:37]: Are we going to update this paragraph in followup changes?

File: capstone/target/step-96-2020-1/WEB-INF/appengine-web_copy.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (zhanche-cp) [inline capstone/target/step-96-2020-1/WEB-INF/appengine-web_copy.xml:1]: It seems we already have appengine-web.xml. Why do we need a copy here?

**Comment by magwitz** [comment]
> Hmm looks like the conflicts are still here -- were you able to figure out how to remove those files from your PR?

Here's the link from our 1:1 with instructions: https://stackoverflow.com/a/44355701

Since you need to keep a local version, you can run the following command to prevent it from ever being committed in the future (once you remove it from this PR)
`git update-index --assume-unchanged <path/to/file.txt>`

File: capstone/target/step-96-2020-1/script.js (hunk 1 above)
Comment (derachimso) [reply capstone/target/step-96-2020-1/script.js]: I deleted the whole file, but the target folder seems to contain duplicated that never updated.

**Comment by derachimso** [comment]
All of these comments seem to be on the problematic files from the target folder. I will have to get that out.

File: capstone/target/step-96-2020-1/WEB-INF/appengine-web_copy.xml (hunk 4 above)
Comment (derachimso) [reply capstone/target/step-96-2020-1/WEB-INF/appengine-web_copy.xml]: The whole maven folder seems to have duplicates and Maggie is looking into it. Will leave this here for now.

File: capstone/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,10 +1,44 @@
...
+<!doctype html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>Default Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link href="style.css" type="text/css" rel="stylesheet">
+        <title>Home</title>
+
```
Comment (magwitz) [inline capstone/src/main/webapp/index.html:9]: Please remove the extra empty lines in this file. Makes sense to have one between the two major divs, but the rest seem unnecessary

Comment (derachimso) [reply capstone/src/main/webapp/index.html]: I just did this. Let me know what else I need to do.

**Pull request merged**